run/README.md

```diff
@@ -15,6 +15,7 @@ This directory contains samples for [Google Cloud Run](https://cloud.run). [Clou
 |[Cloud Pub/Sub][pubsub] | Handling Pub/Sub push messages | [<img src="https://storage.googleapis.com/cloudrun/button.svg" alt="Run on Google Cloud" height="30">][run_button_pubsub] |
 |[Cloud SQL (MySQL)][mysql]        | Use MySQL with Cloud Run    |      -        |
 |[Cloud SQL (Postgres)][postgres]  | Use Postgres with Cloud Run |      -        |
+|[Django][django]                  | Deploy Django on Cloud Run  |      -        |
```

Comment:
This doesn't seem to be hyperlinking the way the others are

Reply:
Fixed! Good catch!